admin/database/database.go

```diff
@@ -947,13 +948,14 @@ type ProjectRole struct {
 type OrganizationMemberUser struct {
 	ID              string
 	Email           string
-	DisplayName     string    `db:"display_name"`
-	PhotoURL        string    `db:"photo_url"`
-	RoleName        string    `db:"role_name"`
-	ProjectsCount   int       `db:"projects_count"`
-	UsergroupsCount int       `db:"usergroups_count"`
-	CreatedOn       time.Time `db:"created_on"`
-	UpdatedOn       time.Time `db:"updated_on"`
+	DisplayName     string         `db:"display_name"`
+	PhotoURL        string         `db:"photo_url"`
+	RoleName        string         `db:"role_name"`
+	ProjectsCount   int            `db:"projects_count"`
+	UsergroupsCount int            `db:"usergroups_count"`
+	CreatedOn       time.Time      `db:"created_on"`
+	UpdatedOn       time.Time      `db:"updated_on"`
+	Attributes      map[string]any `db:"attributes"`
```

Comment:
nit: move `Attributes` higher in the type (the fields at the end of the type are computed fields)

```suggestion
	RoleName        string         `db:"role_name"`
	Attributes      map[string]any `db:"attributes"`
	ProjectsCount   int            `db:"projects_count"`
	UsergroupsCount int            `db:"usergroups_count"`
	CreatedOn       time.Time      `db:"created_on"`
	UpdatedOn       time.Time      `db:"updated_on"`
```